stellargraph/mapper/full_batch_generators.py

```diff
@@ -117,7 +117,7 @@ def __init__(
             self.use_sparse = sparse
 
         # Get the features for the nodes
-        self.features = G.node_features(self.node_list)
+        self.features = G.node_features_tensors(self.node_list)
```

Comment:
Does this get computed eagerly and realised into memory here? (This is just a general question, and doesn't/shouldn't need changes here.)

Reply:
AFAIK in eager mode yeah that get realised into memory here, but in graph mode I think these stay as symbolic tensors and every time a keras model accesses them it runs the feature lookup code. 

I think in the case of `model.fit(..., epochs=something > 1)` the tensorflow graph optimizer recognizes that this op's output is constant and realizes the symbolic tensor as a constant tensor for the entirety of the `.fit` call. So tensorflow only does the feature lookup at the start of each `fit` or `predict` call.

Reply:
Cool, the latter sounds like exactly what we want 🏎